issueTemplate.md

```diff
@@ -0,0 +1,17 @@
+**WARNING: Issue will be closed if it does not follow these guidelines.
+# Descriptive Title of Issue Being Raised
+<!-- Summarize the feature, bug, or other issue in the title. -->
+# Add Detailed Comment with More Information
+<!-- Expand upon the detailed title with more in-depth information to help people understand the issue. -->
+# Select Labels Based on Issue
```

Comment:
Is this a necessary header? It is a step, but does this information need to be included in the issue description?

This and many other headers afterwards seem more like instructions for handling issues than headers that form a template for an issue.

Reply:
Some of them do look like instructions for handling issues, but I was basing the headers off of how I thought someone reviewing the issue would like to see them to have the most information and easiest time understanding the issue. If you feel this is unnecessary, I can edit or delete it accordingly.

Reply:
These issues are for you to decide how to handle. Talk with your leadership and figure out what design you want to follow! I'm just pointing out things that may perhaps be bad practice or inconsistent.

Reply:
My team has met and we think this design works. @gkapfham, what do you think about this?

Reply:
@Michionlion, My group has just updated the Issue Template to follow your suggestions. @Lancasterwu and @thomad74